cmake/FlowCommands.cmake

```diff
@@ -133,17 +133,8 @@ function(strip_debug_symbols target)
     DEPENDS ${target}
     COMMENT "Stripping symbols from ${target}")
   add_custom_target(strip_only_${target} DEPENDS ${out_file})
-  if(is_exec AND NOT APPLE)
-    add_custom_command(OUTPUT "${out_file}.debug"
-      DEPENDS strip_only_${target}
-      COMMAND objcopy --verbose --only-keep-debug $<TARGET_FILE:${target}> "${out_file}.debug"
-      COMMAND objcopy --verbose --add-gnu-debuglink="${out_file}.debug" "${out_file}"
-      COMMENT "Copy debug symbols to ${out_name}.debug")
-    add_custom_target(strip_${target} DEPENDS  "${out_file}.debug")
```

Comment:
Why `"${out_file}.debug"` is no longer needed?

Reply:
Found a use case for these files that I was unaware of. I'll revert this change.